src/ansys/fluent/core/solver/flobject.py

```diff
@@ -493,7 +493,6 @@ def _show_warning(message, category, *args, **kwargs):
 
 
 warnings.showwarning = _show_warning
-warnings.simplefilter("default", category=DeprecatedSettingWarning)
```

Comment:
`DeprecationWarning` from a module is ignored by default, but `FutureWarning` (recommeded as user-facing warning) is not.